Assets/_Scripts/Character/StatusAilment.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public abstract class StatusAilment {
+	public string name;
+	public int duration;
+	public int level;
+	public BattlePhase phase;
+
+	public virtual void DecrementDuration() {
+		duration--;
+	}
+
```

Comment:
Do you think we would need a IsValid() function? I.e. you can't get poisoned twice, bosses can't get hit by percentage health effects, etc